### ISSUE TITLE
Initialising with a set of evaluated pairs (x0, y0)

### ISSUE THREAD

**Issue by amirsani** [EpistasisLab/tpot#1033]
Great AutoML tool. Thank you!

I like to scan a mesh of parameter combinations before HPO. How can I seed TPOT's GA/GP with these evaluations?

Is there an existing approach to this? Parameters for this are usually of the form,
```python
x0
list, list of lists or None
Initial input points.

y0
list, scalar or None
Evaluation of initial input points.
```

**Comment by weixuanfu** [comment]
Do you mean [TPOT API](https://epistasislab.github.io/tpot/api/)?

**Comment by amirsani** [comment]
The API doesn't seem to offer this. It would be nice. Something like this,

```
tpot = TPOTClassifier(generations=5, population_size=10,
                                    x0=params,
                                    y0=score_evaluations)
```

**Comment by weixuanfu** [comment]
Hmm, I don't understand what x0 and y0 stand for? Could you explain a little bit more?

**Comment by amirsani** [comment]
Sure. If I have some parameter combinations for a specific algorithm, such as XGBoost, x0_xgboost, and their resulting evaluations, y0_xgboost, I would like to submit these to the GA to help improve the search over xgboost. I could have the same over a set of sklearn algorithms. It would be useful to use these to improve the TPOT search. 

It could be submitted as a dictionary,
```
evals = {"model":xgbclassifier,
             "x0":list of dictionary_of_parameter_combinations,
             "y0":list of scores}
```

Then I could call tpot like so,
```
tpot = TPOTClassifier(generations=5, 
                                    population_size=10,
                                    evals=evals)
```

**Comment by weixuanfu** [comment]
Hmm, about x0, I think it is about `config_dict` parameter, please check [this link](https://epistasislab.github.io/tpot/using/#customizing-tpots-operators-and-parameters) for example usage. 

And [here](https://github.com/EpistasisLab/tpot/blob/master/tpot/config/classifier.py#L101-L108) is xgboost's configuration in default TPOTClassifier. 

For y0, I think it is about `scoring` parameter but TPOT does not support a list of scores for now.

**Comment by amirsani** [comment]
Thank you, this would work well.
